addons/mail/models/mail_thread.py

```diff
@@ -2236,6 +2236,26 @@ def _message_create(self, values_list):
             self = self.with_context(ctx)
         return self.env['mail.message'].create(create_values_list)
 
+    def _message_get_default_subject(self, record_name=None):
```

Comment:
As we already have ``_message_compute_author`` and ``_message_compute_parent_id``, maybe we could name this one ``_message_compute_subject`` (and move it before ``_message_create``) ?